browser/resources/settings/brave_overrides/index.ts

```diff
@@ -9,6 +9,8 @@
 // where it is not consistant due to "network" responses) and
 // optimized (rollup controls the order in which modules are executed).
 
+import 'chrome://resources/brave/br_elements/br_toolbar/br_toolbar.js'
```

Comment:
```suggestion
import '//resources/brave/br_elements/br_toolbar/br_toolbar.js'
```

Reply:
Fixed in [14403c9](https://github.com/brave/brave-core/pull/26561/commits/14403c9f9a9a64e6679ec7a5b66890f3200ee5ac)